modules/andes-core/client/src/main/java/org/wso2/andes/client/XAConnectionImpl.java

```diff
@@ -43,29 +46,47 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(XAConnectionImpl.class);
 
     /**
+     * After this delay(in seconds), the connection will be closed even if a commit or rollback is not received. This
+     * is done to avoid stale connections to broker.
+     */
+    private static final int CONNECTION_CLOSE_TIMEOUT = 60;
```

Comment:
Can we test the existing setup and see the average time it takes for a commit to get received? We should be able to get an idea from that.